### DISCUSSION TITLE
Question about map

### DISCUSSION THREAD

**Discussion by Tomanic-glitch** [ngocjohn/vehicle-status-card#39] [Q&A] [answered]
Hi @ngocjohn, I was wondering if the pulsating circle on the map portion of the vehicle status card does have any meaning or function?

![map_circle](https://github.com/user-attachments/assets/d53c31c8-5970-4c25-96f7-7654465a626c)

**Comment by ngocjohn** [comment, marked as answer]
Hi @Tomanic-glitch, It tries to fetch addresses from coordinates, the default is set to openstreetmap and there are daily limits for fetch api, if the limit is reached it returns empty data. Or the address cannot be fetched. I will probably hide this animation if there is no data to make it less confusing. 😄

**Comment by Tomanic-glitch** [comment]
Ok got it, many thanks for the explanation! 😄
